modules/exploits/windows/http/easy_fileshare_ws_bof.rb

```diff
@@ -0,0 +1,104 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = GreatRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'    => 'Easy File Sharing Web Server - USERID Remote Buffer Overflow',
+      'Description'  => %q{
```

Comment:
How about:

This module exploits the buffer used to store the USERID component of a HTTP request. A specially crafted request will overflow this buffer resulting in arbitrary code execution with the permissions of the user running the daemon.